crates/task-impls/src/consensus/proposal_helpers.rs

```diff
@@ -487,9 +488,14 @@ async fn publish_proposal_from_commitment_and_metadata<TYPES: NodeType>(
     // FIXME - This is not great, and will be fixed later.
     // If it's > July, 2024 and this is still here, something has gone horribly wrong.
     let cnm = commitment_and_metadata
-        .clone()
+        .take()
```

Comment:
So then should we remove the below setting commit_and_metadata to None explicitly?